### PR TITLE
Tabs: Design sync

### LINKED ISSUE #606: Design-Sync Tabs
https://share.goabstract.com/3d737ae4-3e5c-409f-8524-69ab5acd8fb0?collectionLayerId=62ead9c2-f66f-40e2-8b35-851fda4fc592&mode=design

* Remove obsolete tab variants/themes
* Remove variant and theme prop. There is only one style anymore.
* Overflow for small screens
* Update style

### PR DESCRIPTION
**Pull request by ritikamotwani**
- Removed the coloured theme
- Added an example with badge component in the tabs
- Added overflow fade effect

Fixes: #606 

The overflow effect has been QC'd

### CODE REVIEW

**Comment by ritikamotwani** [comment]
Overflow effect taken from https://ant.design/components/tabs/ and https://codepen.io/ramiro-ruiz/pen/YzKVOGK

**Comment by rosslo** [comment]
Hi @ritikamotwani , have you checked whether the removal of 'coloured' breaks the profile page's tab or not?
image


It looks like not a coloured tab, however, it is in the codebase. 😅 

image

**Review by rosslo — COMMENTED** [review]
Should we remove `.horizontal-tabs-list.colored` style from `TabsStyle.ts`?

File: src/Display/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -58,7 +53,6 @@ export const Tabs: Tabs = ({
                     `tab-${tabLabel}`,
                     { active: activeTabOrIndex === tabLabel },
                     `${alignment}-tab`,
-                    `${variant}`,
```
Comment (rosslo) [inline src/Display/Tabs/Tabs.tsx:61]: This might be a breaking change. 
image

**Comment by ritikamotwani** [comment]
@rosslo https://gitlab.glints.com/glints/glints-dst/-/merge_requests/2473 removed the variant prop. Could you review this as well. Will merge it first!

**Comment by ritikamotwani** [comment]
> colored

Right! Removed

**Comment by ritikamotwani** [comment]
@rosslo updated this!

**Review by rosslo — APPROVED** [review]

**Pull request merged**